magefiles/magefile.go

```diff
@@ -214,7 +214,7 @@ func (ci CI) TestE2E() error {
 func RunE2ETests() error {
 	cwd, _ := os.Getwd()
 
-	return sh.RunV("ginkgo", "-p", "--timeout=90m", fmt.Sprintf("--output-dir=%s", artifactDir), "--junit-report=e2e-report.xml", "--v", "--no-color", "--label-filter=$E2E_TEST_SUITE_LABEL", "./cmd", "--", fmt.Sprintf("--config-suites=%s/tests/e2e-demos/config/default.yaml", cwd), "--generate-rppreproc-report=true", fmt.Sprintf("--rp-preproc-dir=%s", artifactDir))
+	return sh.RunV("ginkgo", "-p", "--timeout=90m", fmt.Sprintf("--output-dir=%s", artifactDir), "--junit-report=e2e-report.xml", "--label-filter=$E2E_TEST_SUITE_LABEL", "./cmd", "--", fmt.Sprintf("--config-suites=%s/tests/e2e-demos/config/default.yaml", cwd), "--generate-rppreproc-report=true", fmt.Sprintf("--rp-preproc-dir=%s", artifactDir))
```

Comment:
why was the `--no-color` flag removed? it makes the output [not easy to read when it's archived](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/297/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1614884687627423744/artifacts/redhat-appstudio-e2e/redhat-appstudio-e2e/build-log.txt)

Reply:
Zoran claims that `it'll help with finding the issues faster` https://redhat-internal.slack.com/archives/C02FANRBZQD/p1673604456367239?thread_ts=1673536652.590219&cid=C02FANRBZQD
But I agree with Pavel... IMO it makes the output hard to read (unless @zregvart has some secret trick in his sleeve...)
When I wanted to see it in color, I downloaded the input and then I `cat` that file in my terminal, which did the coloring, but this is not good UX :-D
I've tried to search for some browser extensions that would do this coloring directly in browser, but I wasn't happy with any of them.

Reply:
My assumption is that the primary interaction is using prow web interface which supports colors. Color there helps to distinguish the output better. For example on [this](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/297/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1613991638235877376) I can see the failure immediately, and if I click on `Show all hidden lines` it is much easier to spot individual scenario failures. I was not thinking that someone would download the raw logs and process them locally. Piping to `cat` is an option, as is `less -R` or using an text editor that supports ANSI escape codes - I think most should natively or via plugins/extensions.

Reply:
> For example on [this](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/297/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1613991638235877376) I can see the failure immediately, and if I click on Show all hidden lines it is much easier to spot individual scenario failures.

Other option is to `Show all hidden lines` and then search for "[failed", which works quicker and fine despite having the output coloured or not (IMO). But I won't hold this PR because of this small thing, which can be reverted easily later (if needed). Especially not today, when one has to be super lucky just to claim a cluster for testing the changes.